### ISSUE TITLE
[new feature] What about RSS ?

### ISSUE THREAD

**Issue by rmdes** [Em-AK/makesense_jobs#42]
Any chance to be able to subscribe to the makesense-jobs website using RSS ?

I just want to get an alert with new jobs when they come up for example. 

https://fr.wikipedia.org/wiki/RSS

**Comment by Em-AK** [comment]
Yes this feature has been a recurrent request. And it is not too complex to implement.
